code/src/components/StartGame.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { game, startGame } from 'reducers/game';
+import styled from 'styled-components/macro';
+
+const StyledForm = styled.form`
+display: flex;
+flex-direction: column;
+align-items: center;
+`
+const StyledInput = styled.input`
+font-family: 'Ubuntu Mono', monospace;
+background-color: #FEF5BD;
+border: none;
+padding: 7px 5px 7px 10px;
+outline: none;
+font-size: 1rem;
+color: #000;
```

Comment:
Really like the styling of the whole game and the old school adventure game-feel of it